apps/web/src/app/globals.css

```
@@ -1,4 +1,5 @@
 @import 'tailwindcss';
+@source "../../../../packages/ui/src";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)